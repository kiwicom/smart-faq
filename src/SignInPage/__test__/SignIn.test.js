// @flow

import * as React from 'react';
import { shallow } from 'enzyme';

import SignIn from '../';

const history = {
  push: () => {},
};
describe('SignIn', () => {
  const component = shallow(<SignIn history={history} />);
  it('should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
