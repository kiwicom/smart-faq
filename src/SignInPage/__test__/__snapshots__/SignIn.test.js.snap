// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn should match snapshot 1`] = `
<Route
  render={[Function]}
/>
`;

exports[`SignIn should match snapshot with expired message 1`] = `
<Route
  render={[Function]}
/>
`;
